app/models/search_settings.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: search_settings
+#
+#  id           :integer          not null, primary key
+#  community_id :integer          not null
+#  main_search  :string(255)      default("KEYWORD"), not null
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+
+class SearchSettings < ActiveRecord::Base
+  attr_accessible(
```

Comment:
@sktoiva @rap1ds thanks!
I can remove those. In the next step / branch I'm planning to add service API access point to MarketplaceService::API (unless you have better suggestions). 

Currently, I feel that a separate Search Settings Service for one flag is too much (and the actual new "search service" is called Listing Index Service, which seems to be something else than this flag for search bar type).